src/Elcodi/Store/PaymentBridgeBundle/Services/PaymentBridge.php

```diff
@@ -172,9 +172,13 @@ public function getExtraData()
                     ->getPurchasable()
                     ->getName();
 
+                /*
+                 * Elcodi stores amounts in integer cents (10USD = 1000)
+                 * but Paypal expects decimal (10.00USD)
+                 */
                 $orderLineArray['amount'] = $orderLine
                     ->getAmount()
-                    ->getAmount();
+                    ->getAmount() / 100;
```

Comment:
@mmoreram actually this is Paypal related (we should call this class PaypalBridge or something like that).
However in the PayPal case, it *always* expects amounts in decimal (like `10.50`) so it's up to the bridge to convert them (in this case with a simple / 100).
Remember that this is Elcodi specific (it is just exposing amount stores in the cart lines, which are `Money` VOs). We *do* know that those are represented in cents.
Suggestions? (without this fix we will go to the paypal test gateway page with the amounts multiplied by 100!)
@xphere the decimal definition depends on https://github.com/sebastianbergmann/money/blob/master/src/Currency.php (a private member). Maybe we can use some reflection trick to read those settings (or just configure them somewhere else)